src/message/main.scss

```diff
@@ -9,11 +9,6 @@
     @include box-sizing;
     @include message-bounding();
 
-    .#{$css-prefix}message-wrapper {
```

Comment:
It is used in `src/message/toast.jsx` line 79. We'd better not delete any styles that are not written by ourselves

Reply:
It actually isn't used in toast because the `.next-message-wrapper` is not nested inside of a `.next-message`. I checked that the styles are not affected.

Reply:
yes, you're right